apps/studio/src/lib/db/clients/utils.ts

```diff
@@ -39,6 +42,34 @@ export function joinQueries(queries) {
   return results.join("")
 }
 
+export async function whichTool({ toolName }: { toolName: string }) {
+  const command = `${platformInfo.isWindows ? 'where' : 'which'}`
+
+  return new Promise<string>((resolve, reject) => {
+    const proc = spawn(command, [toolName], { shell: true });
```

Comment:
See -- we're properly using spawn here to run a command, we should use spawn in the other method also. Maybe make a nice spawn wrapper so you don't have to rely on proc.stdout.on and such